File: challenges/codeeval/easy/067/juanescobar.php
<?php

    $fh = fopen($argv[1], "r");
    while (true) {
        $test = trim(fgets($fh));
        # break loop if $test is an empty line
        if ($test == "")
            break;
        # $test represents the test case, do something with it
        echo hexdec($test)."\n";

    }

?>

File: challenges/codeeval/easy/100/juanescobar.c
#include <stdio.h>

int main(int argc, const char * argv[]) {
    FILE *file = fopen(argv[1], "r");
    char line[1024];
    while (fgets(line, 1024, file)) {
        if (atoi(line) % 2 == 0) {
            printf("%d\n", 1);
        } else {
            printf("%d\n", 0);
        }
    }
    return 0;
}
